### PR TITLE
fix: issue with collection hooks

### PR DESCRIPTION
**Pull request by jdgjsag67251**
This package performs multiple `find` operations. This can cause issues when the [Meteor collection hooks package](https://github.com/Meteor-Community-Packages/meteor-collection-hooks) is installed; since this can add restrictions on the selectors which may prevent updates from being sent.
This PR just adds a check if the `direct` key is available, if so it will use that for all internal `find` operations.

### CODE REVIEW

File: lib/redis/RedisSubscriptionManager.js [hunk 1]
```diff
@@ -164,6 +164,9 @@ class RedisSubscriptionManager {
                     debug(
                         `[RedisSubscriptionManager] Exception while processing event: ${e.toString()}`
                     );
+                    Meteor._debug(
```
Comment (jdgjsag67251) [inline lib/redis/RedisSubscriptionManager.js:167]: Previously these were just swallowed when debug was not turned on

**Review by KoenLav — APPROVED** [review]

**Comment by StorytellerCZ** [comment]
Any chance of fixing the tests?

**Comment by jdgjsag67251** [comment]
BTW, when I tried running the tests locally I ran into a syntax error in Chai because it uses some JS features that are not available in Node v14. I installed `chai@legacy` and it started working.

**Comment by StorytellerCZ** [comment]
Once the tests pass I will release it as a patch.

**Comment by StorytellerCZ** [comment]
@jdgjsag67251 some issues still remain.

**Pull request closed without merging**